src/kbmod_wf/task_impls/reproject_wu.py

```diff
@@ -12,3 +23,289 @@ def reproject_wu(input_wu=None, reprojected_wu=None, logger=None):
+        self.reprojected_wu_filepath = reprojected_wu_filepath
+        self.runtime_config = runtime_config
+        self.logger = logger
+
+        self.overwrite = self.runtime_config.get("overwrite", False)
+        self.search_config = self.runtime_config.get("search_config", None)
+
+        # Default to 8 workers if not in the config. Value must be 0<num workers<65.
+        self.n_workers = np.max(1, np.min(self.runtime_config.get("n_workers", 8), 64))
+
+        # ! Decide if we want these from a runtime_config or exclusively read from the uri file
+        self.patch_corners = self.runtime_config.get("patch_corners", None)
+        self.guess_dist = self.runtime_config.get("guess_dist", None)
+        self.pixel_scale = self.runtime_config.get("pixel_scale", None)
+
+        # ! Decide if we want these from a runtime_config or calculate each time based on self.uri_params
+        self.image_width = self.runtime_config.get("image_width", None)
+        self.image_height = self.runtime_config.get("image_height", None)
+
+        self.uri_params = self._get_params_from_uri_file(uri_file=self.uri_file)
+
+        self.pixel_scale = self.uri_params["pixel_scale"]
+        self.patch_size = self.uri_params["patch_size"]
+
+        # handle image dimensions
+        if self.image_width == None or self.image_height == None:
+            if "patch_size" not in self.uri_params:
+                raise KeyError(
+                    f"Must supply image dimensions (image_width, image_height) or #patch_size= must be in a specified URI file."
+                )
+            if self.pixel_scale == None:
+                raise KeyError(
+                    f"When patch pixel dimensions are not specifified, the user must supply a pixel scale via the command line or the uri file."
+                )
+
+        self.image_width, self.image_height = self._patch_arcmin_to_pixels(
+            patch_size_arcmin=self.patch_size,
+            pixel_scale_arcsec_per_pix=self.pixel_scale,
+        )
+
+        self.point_on_earth = EarthLocation(1814303.74553723, -5214365.7436216, -3187340.56598756, unit="m")
+
+    def reproject_workunit(self):
+        self.logger.debug(f"Creating WCS from patch")
+
+        # Create a WCS object for the patch. This will be our common reprojection WCS
+        patch_fits_name = f"patch.fits"
+        if "patch_id" in self.uri_params:
+            patch_fits_name = f"patch_{self.uri_params['patch_id']}.fits"
+
+        # ! TODO: Need to figure out what to do with args.result_dir here. Pass `filename` as an parsl.output?
+        # ! Also, the _create_wcs_from_corners function doesn't make use of all the self.<variables> available to it.
+        patch_wcs = self._create_wcs_from_corners(
+            self.patch_corners,
+            self.image_width,
+            self.image_height,
+            pixel_scale=self.pixel_scale,
+            filename=os.path.join(args.result_dir, patch_fits_name),
+        )
```

Comment:
@wilsonbb This comment is regarding lines 103:115. I don't recall what we do with `patch.fits`. If we need to keep this intermediate file, then I can modify the workflow so that it's passed in as a parameter to the python app, so that parsl can track and make sure that it is created. 

But I also don't recall how/if this is used. Do we need to keep the file around, or do we just need to keep the result `patch_wcs`?

Reply:
I would say that this can be removed. I don't think this was ever ultimately used, and by storing the patch corners in the workflow files (such as the '.lst' file), we could recalculate later if needed

Reply:
Updated this so that it's not writing out the file.